src/Command/SchemaLoadCommand.php

```diff
@@ -59,9 +59,18 @@ public function execute(InputInterface $input, OutputInterface $output)
 
         $dbmanager = new DatabaseManager();
         $databaseConfig = $dbmanager->getDatabaseConfig($url);
-        $dbname = $databaseConfig->getConnectionConfig('default')->getDatabaseName();
+        $config = $databaseConfig->getConnectionConfig('default');
```

Comment:
@congpeijun I think it's more proper to throw exception from the database-manager::getPdo method